contracts/peripheral/Strategies/Curve.sol

```diff
@@ -16,10 +16,16 @@ interface ICurvePool {
 interface ICurvePoolFactory {
     function get_coins(address _pool) external view returns (address[4] memory);
 
-    function deploy_plain_pool(string memory _name, string memory _symbol, address[4] memory _coins, uint256 _A, uint256 _fee) external returns (address);
+    function deploy_plain_pool(
```

Comment:
Can you remove the functions found in lines 19-29 as they're used in the contract.

Reply:
i need them for tests